src/models/focus.js

```diff
@@ -99,9 +100,19 @@ nv.models.focus = function(content) {
             brush
                 .x(x)
                 .on('brush', function() {
-                    onBrush();
+                    onBrush(syncBrushing);
                 });
 
+            if (!syncBrushing) {
+                brush.on('brushend', function () {
+                    var extent = brush.empty() ? x.domain() : brush.extent();
+                    if (Math.abs(extent[0] - extent[1]) <= 1) {
+                        return;
+                    }
+                    dispatch.onBrush(extent);
+                });
+            }
```

Comment:
This block gets flipped on when you set syncBrushing to false, but then you can't change it back ever again... why not always handle brushend and inside there just check the value of syncBrushing so that it handles it regardless?